optimum/utils/__init__.py

```diff
@@ -12,22 +12,60 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 import importlib.util
+import inspect
+from contextlib import contextmanager
+
+from packaging import version
 
 
 CONFIG_NAME = "config.json"
 
 _onnxruntime_available = importlib.util.find_spec("onnxruntime") is not None
 _pydantic_available = importlib.util.find_spec("pydantic") is not None
+_accelerate_available = importlib.util.find_spec("accelerate") is not None
 
 
 def is_onnxruntime_available():
+    try:
+        # Try to import the source file of onnxruntime
```

Comment:
```suggestion
        # Try to import the source file of onnxruntime - if you run the tests from `tests` the function gets 
        # confused since there a folder named `onnxruntime` in `tests`. Therefore, `_onnxruntime_available`
        # will be set to `True` even if not installed.
```